deploy-move/FPIInitParams.ts

```diff
@@ -0,0 +1,10 @@
+const oftMetadata = {
+    token_name: 'Frax Price Index',
+    token_symbol: 'FPI',
+    icon_uri: '',
+    project_uri: '',
+    sharedDecimals: 6,
+    localDecimals: 6,
+}
+
+export { oftMetadata }
```

Comment:
Will the `oftMetadata` variable be overwritten by the other `oftMetadata` exports?

Reply:
No. Need to specify correct file while deploying.